.github/workflows/run_tests.yml

```diff
@@ -1,6 +1,6 @@
 on:
   schedule:
-  - cron: "0 5 * * *" # 12pm EST
+  - cron: "0 5,17 * * *" # 12am and 12pm EST
```

Comment:
This will actually be 1am and 1pm during daylight savings time right?  That's probably fine though

Reply:
yes. 1 and 13 EDT